website/docs/tutorials/08-cairo-1-support/03-cheatcodes/deploy_contract.md

```diff
@@ -0,0 +1,33 @@
+# `deploy_contract`
+
+```cairo
+fn deploy_contract(contract: felt252, calldata: Array::<felt252>) -> 
+Result::<felt252, RevertedTransaction>
+```
+
+Declares and deploys a contract given its name defined in the [protostar.toml](../../04-configuration-file.md) configuration
+file.
+
+```cairo title="Example"
+use result::ResultTrait;
+use array::ArrayTrait;
+
+fn my_function() {
+
+    let mut calldata = ArrayTrait::new();
+    calldata.append(10);
+    calldata.append(11);
+    calldata.append(12);
+
+    let address = deploy_contract('mycontract', calldata).unwrap();
```

Comment:
NIT: we could use `constructor_calldata` for clarity.

Reply:
I left calldata for consistency with other tutorials